controller/array_action/array_mediator_xiv.py

```diff
@@ -131,18 +138,21 @@ def _generate_snapshot_response(self, cli_snapshot):
                         cli_snapshot.wwn,
                         cli_snapshot.name,
                         self.endpoint,
-                        volume_id=cli_snapshot.copy_master_wwn,
+                        volume_id=self._get_copy_source_object_wwn(cli_snapshot),
                         is_ready=True,
                         array_type=self.array_type)
 
-    def get_volume(self, volume_name, pool=None):
-        logger.debug("Get volume : {}".format(volume_name))
+    def _get_cli_object_by_name(self, volume_name):
         try:
-            cli_volume = self.client.cmd.vol_list(vol=volume_name).as_single_element
+            return self.client.cmd.vol_list(vol=volume_name).as_single_element
         except xcli_errors.IllegalNameForObjectError as ex:
             logger.exception(ex)
             raise array_errors.IllegalObjectName(ex.status)
 
+    def get_volume(self, volume_name, pool=None):
+        logger.debug("Get volume : {}".format(volume_name))
+        cli_volume = self._get_cli_object_by_name(volume_name)
```

Comment:
Does the function return a CLI object or CLI volume?

Reply:
it's either a cli volume or a [cli snapshot](https://github.com/IBM/ibm-block-csi-driver/blob/ceb5060d18fa53856290428ec033e0d5e8a7a519/controller/array_action/array_mediator_xiv.py#L313), and the current common name for volume and snapshot, across this project, is "object"